### DISCUSSION TITLE
Welcome to nattermob.dev Discussions!

### DISCUSSION THREAD

**Discussion by raae** [raae/nattermob.dev#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swabah** [comment]
I created a social web application using React JS. When sharing an article to social media, I can’t see the preview. A YouTube video tutorial example didn’t work for me.
